keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifact/ArtifactListener.kt

```diff
@@ -30,8 +32,19 @@ class ArtifactListener(
       .forEach { korkArtifact ->
         val artifact = korkArtifact.toDeliveryArtifact()
         if (artifactRepository.isRegistered(artifact.name, artifact.type)) {
-          val version = "${korkArtifact.name}-${korkArtifact.version}"
-          val status = artifactStatus(korkArtifact)
+          val version: String
+          val status: ArtifactStatus
+          when {
+            korkArtifact.isDeb() -> {
+              version = "${korkArtifact.name}-${korkArtifact.version}"
+              status = debStatus(korkArtifact)
+            }
+            korkArtifact.isDockerImage() -> {
+              version = korkArtifact.name + ":" + korkArtifact.version
```

Comment:
nit: Any reason not to use the string template like on line 39?

Reply:
Updating, good catch.